admin/client/src/components/ListGroup/LIstGroup.scss

```diff
@@ -0,0 +1,96 @@
+// List-group, based on Bootstraps list-group
+// TODO split out campaign specific css from generic list-group
+
+.list-group {
+  margin-left: -$spacer-y;
+  margin-right: -$spacer-y;
+  border-bottom: 1px solid $border-color-light;
+  margin-bottom: $spacer-y*.75;
+}
+
+.list-group-item {
+	padding-left: $spacer-y;
+	padding-right: $spacer-y;
+	min-height: 64px;
+	cursor: pointer;
+	text-decoration: none;
+
+	&:first-child {
+		border-top: none;
+	}
+	&:hover {
+		text-decoration: none;
+
+		.list-group-item--has-links {
+			display: block;
+			opacity: 1;
+		}
+	}
+
+	&.active {
+		background-color: $brand-primary;
+		color: #FFF;
+		opacity: 1;
+
+		.list-group-item--has-links,
+		.list-group-item__status {
+			color: #FFF;
+			opacity: 1;
+		}
+	}
+}
+
+.list-group-item__heading {
```

Comment:
Bootstrap has a standard `list-group-item-heading` class we should probably use.